modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -2864,14 +2859,15 @@ else if (msg instanceof SnapshotFilesFailureMessage) {
             try {
                 task.partsLeft.compareAndSet(-1, partsCnt);
 
-                File cacheDir = U.resolveWorkDirectory(task.dir.toString(),
-                    Paths.get(rmtDbNodePath, cacheDirName).toString(),
-                    false);
+                File cacheDir = new File(storeMgr.workDir(), cacheDirName);
```

Comment:
Use the _FilePageStoreManager.cacheWorkDir_

Reply:
Done, thanks